src/change_stream.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
this comment should not be necessary anymore along with its instances in the other affected files